pkg/storage/chunk/client/aws/s3_storage_client.go

```diff
@@ -117,6 +118,7 @@ func (cfg *S3Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	cfg.SSEConfig.RegisterFlagsWithPrefix(prefix+"s3.sse.", f)
 
 	f.DurationVar(&cfg.HTTPConfig.IdleConnTimeout, prefix+"s3.http.idle-conn-timeout", 90*time.Second, "The maximum amount of time an idle connection will be held open.")
+	f.DurationVar(&cfg.HTTPConfig.Timeout, prefix+"s3.http.timeout", 5*time.Minute, "Timeout specifies a time limit for requests made by s3 Client.")
```

Comment:
This is a nice addition but we shouldn't set it to be on by default. Can you make the default zero?

Reply:
thanks. 👌,done.